app/views/not_subscribed_to_cds.scala.html

```diff
@@ -32,33 +30,20 @@
 
 @layout(pageTitle = Some(messages("cf.not-subscribed-to-cds.detail.title")), help = false) {
     @h1(msg = "cf.not-subscribed-to-cds.detail.heading")
-    @h2(msg = "cf.not-subscribed-to-cds.detail.how-to-get-access-info")
+    @h2("cf.not-subscribed-to-cds.detail.already-subscribed-to-cds", id = Some("already-subscribed-heading"))
 
-    @p(
-        message = "cf.not-subscribed-to-cds.detail.eori-info",
-        tabLink = Some(hmrcNewTabLink(
-                NewTabLink(
-                    language = Some(messages.lang.toString),
-                    classList = Some("govuk-link"),
-                    href=Some(appConfig.registerCdsUrl),
-                    text=messages("cf.not-subscribed-to-cds.detail.eori-link")
-                )
-            )
-        )
-    )
+    <p class="govuk-body">
+        @messages("cf.not-subscribed-to-cds.detail.already-subscribed-to-cds-guidance-text")
+    </p>
 
-    @p(
-        message = "cf.not-subscribed-to-cds.detail.get-access-info",
-        tabLink = Some(hmrcNewTabLink(
-                NewTabLink(
-                    language = Some(messages.lang.toString),
-                    classList = Some("govuk-link"),
-                    href=Some(appConfig.subscribeCdsUrl),
-                    text=messages("cf.not-subscribed-to-cds.detail.get-access-link")
-                )
-            )
-        )
-    )
+    @h2(messages("cf.not-subscribed-to-cds.details.subscribe-to-cds"), id = Some("subscribed-to-cds-heading"))
 
-    @p("cf.not-subscribed-to-cds.detail.process-application-info")
+    <p class="govuk-body">
```

Comment:
Will probably need to add this class `govuk-!-margin-bottom-7` to the P tag for extra spacing below